app/src/verifier.h

```diff
@@ -362,17 +363,106 @@ namespace scitt::verifier
         }
         else if (phdr.issuer.has_value())
         {
-          // IETF SCITT claim
-          key = process_ietf_profile(
-            phdr, tx, current_time, resolution_cache_expiry, configuration);
-
-          try
+          if (phdr.issuer->starts_with("did:x509"))
           {
-            cose::verify(data, key);
+            // IETF SCITT did:x509 claim
+            if (phdr.x5chain.has_value())
+            {
+              // NB: In later revisions of SCITT, x5chain is unprotected, and
```

Comment:
Please move this block to a separate method for the consistency sake and to make it a bit more friendly on the eyes, to something like `process_did_x509` or similar.

Reply:
Yes, absolutely.